onto-viewer-config-loader/src/main/java/org/edmcouncil/spec/ontoviewer/configloader/configuration/service/YamlFileBasedConfigurationService.java

```diff
@@ -170,8 +166,34 @@ private String loadConfigFromLocalFilesOrDefaultConfigIfNotExist(Path configPath
       try (Stream<Path> configFilePathsStream = Files.walk(configPath, FileVisitOption.FOLLOW_LINKS)) {
         Set<Path> configFilePaths = configFilePathsStream.collect(Collectors.toSet());
         Set<String> notExistingFiles = getNotExistingAndEmptyFiles(configFilePaths);
+
+        if (notExistingFiles.size() == CONFIG_FILES_NAMES.size()){
+          LOGGER.error("The configuration specified by the user does not contains any config files.");
```

Comment:
This error message also appears when the user does not explicitly specify `Config URL` in the `updateUrl` parameter, but there is a default empty `config` configuration directory.